Packages/ClientRuntime/Sources/Networking/Http/UnknownHttpServiceError.swift

```diff
@@ -5,6 +5,8 @@
 
 /// General Service Error structure used when exact error could not be deduced from the `HttpResponse`
 public struct UnknownHttpServiceError: HttpServiceError, Swift.Equatable {
+    public var _errorCode: String?
```

Comment:
Unrelated to this PR:
I wonder why we prefix all the properties here with an underscore? 🤔 
Especially since there properties are public

Reply:
I don't know the purpose either; in the past I have used underscore on private properties to help keep them separate from a related public property.

When we overhaul our error system...